packages/luna/src/components/TutorCard/TutorCardV1.tsx

```diff
@@ -0,0 +1,216 @@
+          className="tw-ellipsis tw-line-clamp-2 tw-text-natural-800"
+        >
+          {about}
+        </Typography>
+
+        <Link to={profileUrl} className="tw-cursor-pointer">
+          <Typography
+            element="caption"
+            weight="bold"
+            className="tw-ellipsis tw-line-clamp-2 tw-text-natural-800 tw-underline"
+          >
+            {intl("tutors.card.label.read-more")}
+          </Typography>
+        </Link>
+
+        {isTutorStats ? (
+          <div className={cn("tw-flex tw-gap-8 tw-my-[15px]")}>
+            {studentCount ? (
+              <div className="tw-flex tw-flex-col">
+                <Typography
+                  element="caption"
+                  weight="regular"
+                  className="tw-text-natural-800"
+                >
+                  {intl("tutors.card.label.students")}
+                </Typography>
+                <Typography
+                  element="body"
+                  weight="semibold"
+                  className="tw-text-natural-950 tw-leading-4-half"
+                >
+                  {formatNumber(studentCount)}
+                </Typography>
+              </div>
+            ) : null}
+
+            {lessonCount ? (
+              <div className="tw-flex tw-flex-col">
+                <Typography
+                  element="caption"
+                  weight="regular"
+                  className="tw-text-natural-800"
+                >
+                  {intl("tutors.card.label.lessons")}
+                </Typography>
+                <Typography
+                  element="body"
+                  weight="semibold"
+                  className="tw-text-natural-950 tw-leading-4-half"
+                >
+                  {formatNumber(lessonCount)}
+                </Typography>
+              </div>
+            ) : null}
+
+            {rating ? (
+              <div className="tw-flex tw-flex-col">
+                <Typography
+                  element="caption"
+                  weight="regular"
+                  className="tw-text-natural-800"
+                >
+                  {intl("tutors.card.label.rating")}
+                </Typography>
+                <div className="tw-flex tw-flex-row tw-items-center tw-gap-[5px] tw-h-full">
+                  <Typography
+                    weight="semibold"
+                    className="tw-inline-block tw-text-natural-950 tw-text-xs tw-leading-3-half"
+                  >
+                    {rating}
+                  </Typography>
+                  <SStar
+                    width={15}
+                    height={15}
+                    className="[&>*]:tw-fill-warning-500"
+                  />
+                </div>
+              </div>
+            ) : null}
+          </div>
+        ) : null}
+        {topics ? (
+          <div className="tw-flex tw-mt-[15px] tw-gap-2 tw-flex-wrap tw-justify-start">
+            {topics.map((topic, idx, arr) => {
```

Comment:
The logic inside this map is not clean. I think we can make it cleaner. 

Here is a sudo code on how it can be done. 

```tsx
const {displayedTopics, remainingTopicsCount} = useMemo(() => {
 const topicCount = isTutorStats ? 4 : 8;
 const displayedTopics = topics.slice(0, topicCount);
 const totalTopics = topics.legnth
 const remainingTopicsCount = totalTopics < topicCount ? 0 : totalTopics - topicCount;
 return {displayedTopics, remainingTopicsCount}
}, [])

```